test/specs/trailApexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,208 @@
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests', fiveMinutes);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Listening for streaming state changes...', fiveMinutes);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Processing test run', fiveMinutes, false);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify test results are listed on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('Assertion Failed: incorrect ticker symbol');
+    expect(outputPanelText).toContain('Expected: CRM, Actual: SFDC');
+  });
+
+  step('SFDX: Get Apex Debug Logs', async () => {
+    // Run SFDX: Get Apex Debug Logs
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Get Apex Debug Logs', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Getting Apex debug logs', fiveMinutes);
+
+    // Select a log file
+    const quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    await prompt.selectQuickPick('User User - ApexTestHandler');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Get Apex Debug Logs', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Get Apex Debug Logs successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+  });
+
+  step('Replay an Apex Debug Log', async () => {
+    // Run SFDX: Launch Apex Replay Debugger with Current File
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Current File', 1);
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Launch Apex Replay Debugger with Current File successfully ran');
+    if (successNotificationWasFound !== true) {
+      const failureNotificationWasFound = await utilities.notificationIsPresent(workbench, 'You can only run this command with Anonymous Apex files, Apex Test files, or Apex Debug Log files.');
+      if (failureNotificationWasFound === true) {
+        expect(successNotificationWasFound).toBe(false);
+      } else {
+        utilities.log('Warning - Launching Apex Replay Debugger with Current File failed, neither the success notification or the failure notification was found.');
```

Comment:
When I run, the "SFDX: Launch Apex Replay Debugger with Current File" command does not display any notifications, and the "Warning - Launching Apex Replay Debugger with Current File failed, neither the success notification or the failure notification was found" condition was hit.

@CristiCanizales when you run, do you get this? (hit line 157 and get the warning)

If you don't get any notification (like me) I suggest changing the code to:

```
  step('Replay an Apex Debug Log', async () => {
    // Run SFDX: Launch Apex Replay Debugger with Current File
    const workbench = await browser.getWorkbench();
    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Current File', 1);

    // Continue with the debug session
    await browser.keys(['F5']);
    await utilities.pause(1);
    await browser.keys(['F5']);
    await utilities.pause(1);

    // TODO:
    // add expectations here
  });
```

Reply:
@jeffb-sfdc I do get a notification, the failure one. That's why I kept it in this case so when the command fully works I just get rid of the failure check part